Sources/HAP/Base/Info.swift

```diff
@@ -5,13 +5,19 @@ extension Service {
         public let model = GenericCharacteristic<String>(type: .model, permissions: [.read])
         public let name = GenericCharacteristic<String>(type: .name, permissions: [.read])
         public let serialNumber = GenericCharacteristic<String>(type: .serialNumber, permissions: [.read])
+        public let firmwareRevision = GenericCharacteristic<String>(type: .firmwareRevision, permissions: [.read])
 
-        public init(name: String, manufacturer: String = "undefined", model: String = "undefined", serialNumber: String = "undefined") {
-            super.init(type: .info, characteristics: [identify, self.manufacturer, self.model, self.name, self.serialNumber])
+        public init(name: String,
```

Comment:
Can you create a separate PR for this?

Reply:
I have done that for the test case, leaving out firmwareVersion for now..